client/init_test.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 func init() {
-	plogrus.Set(logrus.WarnLevel, &logrus.TextFormatter{ForceColors: true})
+	plogrus.Set(logrus.DebugLevel, &logrus.TextFormatter{ForceColors: true})
```

Comment:
By purpose included in this commit?

Reply:
On purpose, so that we see error in the CI.